lib/matplotlib/axes/_axes.py

```diff
@@ -6903,7 +6905,7 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
         """
 
         h, xedges, yedges = np.histogram2d(x, y, bins=bins, range=range,
-                                           normed=normed, weights=weights)
+                                           normed=density, weights=weights)
```

Comment:
Wouldn't it be "density=density"?

Reply:
no, it's called normed on hist2d side until np 1.15... https://github.com/matplotlib/matplotlib/issues/11070#issuecomment-422411546